server/src/test/java/org/elasticsearch/common/xcontent/support/AbstractFilteringTestCase.java

```diff
@@ -484,70 +484,70 @@ public void testSimpleArrayOfObjectsInclusive() throws Exception {
         testFilter(expected, SAMPLE, singleton("authors.*name"), emptySet());
     }
 
-    public void testSimpleArrayOfObjectsExclusive() throws Exception {
-        final Builder expected = builder -> builder.startObject()
```

Comment:
I have not changed anything in the json, only made it a protected static member.